File: src/test/java/io/jsql/test/OrientdbTest.java
package io.jsql.test;

import com.orientechnologies.orient.core.db.OrientDB;
import com.orientechnologies.orient.core.db.OrientDBConfig;
import com.orientechnologies.orient.core.db.document.ODatabaseDocument;
import com.orientechnologies.orient.core.record.OElement;
import org.junit.After;
import org.junit.Before;
import org.junit.Test;
import org.springframework.boot.autoconfigure.web.ServerProperties;

import static org.junit.Assert.*;

/**
 * Created by 长宏 on 2017/5/3 0003.
 */
public class OrientdbTest {
    @Before
    public void setUp() throws Exception {
        assertEquals(1,1);
    }

    @Test
    public void test1() throws Exception {
        assertEquals(1,1);
    }

    @Test
    public void test2() throws Exception {
        OrientDB orientDB = new OrientDB("embedded:./databases", null, null, OrientDBConfig.defaultConfig());
        ODatabaseDocument document = orientDB.open("db1", "admin", "admin");
        assertEquals(document.getName(),"db1");
    }

    @After
    public void tearDown() throws Exception {
    }

}

File: src/test/java/io/jsql/test/OrientdbTest.kt
/*
 * Java-based distributed database like Mysql
 */

package io.jsql.test

import org.junit.After
import org.junit.Before
import org.junit.Test

import org.junit.Assert.assertEquals

/**
 * Created by 长宏 on 2017/5/3 0003.
 */
class OrientdbTest {
    @Before
    @Throws(Exception::class)
    fun setUp() {
        assertEquals(1, 1)
    }

    @Test
    @Throws(Exception::class)
    fun test1() {
        assertEquals(1, 1)
    }

    @Test
    @Throws(Exception::class)
    fun test2() {
    }

    @After
    @Throws(Exception::class)
    fun tearDown() {
    }

}